src/solver/type_getters.jl

```diff
@@ -574,7 +574,6 @@ function get_sim_info(config::AtmosConfig)
         start_date = epoch,
         t_end = t_end,
     )
-    @show sim.t_end, sim.dt
```

Comment:
Can't we just print this from the root process?

Reply:
This information is already printed in the logs at the start of the run, it should not be printed here. 
```
[ Info: Running on CPUSingleThreaded
┌ Info: Setting up distributed ClimaAtmos run
└   nprocs = 2
┌ Info: Time info:
│   dt = "400secs"
│   t_end = "400secs"
└   floor_n_steps = 1
```